store/migrate_test.go

```diff
@@ -217,6 +217,51 @@ func (d *DBV3) compare(td testdb) bool {
 	return true
 }
 
+type DBV4 struct {
```

Comment:
Probably test for migration to V4 and V5 weren't done so this is causing some mess. For consistency the DBVX name should reflect the db version. So it should be DBV6.